update-common/src/artifacts/update_plan.rs

```diff
@@ -108,6 +111,19 @@ struct RotSignTarget {
     bord: String,
 }
 
+struct HostArtifactSource<'a> {
+    gimlet_host_phase_1: &'a mut HashingNamedUtf8TempFile,
+    cosmo_host_phase_1: &'a mut HashingNamedUtf8TempFile,
+    phase_2: &'a mut HashingNamedUtf8TempFile,
+}
```

Comment:
It turned out to be easier to have a separate struct vs just using the `HostPhaseImageSource` from tufaceous directly